lib/broccoli/merge-trees.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
-
 var upstreamMergeTrees = require('broccoli-merge-trees');
+var heimdall = require('heimdalljs');
+var heimdallLogger = require('heimdalljs-logger');
+var logger = heimdallLogger('ember-cli:merge-trees');
 var EMPTY_MERGE_TREE;
 
+heimdall.registerMonitor('ember-cli-merge-trees', function MergeTreesSchema() {
```

Comment:
since it looks like this monitor isn't used outside of the node created on line 48 it would also be fine to pass this in as a schema in `heimdall.start` instead of as a monitor